src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -93,8 +93,7 @@ function (array $value) {
                         ->always($this->createPositiveIntegerValidatorClosure())
                     ->end()
                 ->end()
-            ->end()
-        ;
+            ->end();
```

Comment:
I'm really not a big fan of that change, I find it much less readable and makes less sense

Reply:
I just run PSR2 as considered to be standard. 
I think it's better to have consistend code than special rule for better readability and writeability.

Reply:
I may be mistaken but I think that's a PHP-CS-Fixer thing not PSR-2, or then the PSR-2 doc needs to be updated as a few samples are using that CS as well

Reply:
I didn't find it in examples (`;` is always preceeded by some char), but you are right, it's in `no_empty_statement [@Symfony]`, my bad.

Reply:
Well, it actually from PHP_CodeSniffer: `PHP_CodeSniffer\Standards\PSR2\Sniffs\Methods\FunctionCallSignatureSniff.SpaceAfterCloseBracket`


![image](https://user-images.githubusercontent.com/924196/33051853-3f9d7354-ce6c-11e7-8a2e-2bdd2fc0859e.png)